docs/DeveloperGuide.md

```diff
@@ -557,131 +565,121 @@ Use case ends.
 * 1d. User requests to add a tag that is already present or delete a tag that is absent.
     * 1d1. `PropertyWhiz` warns the user about the tag's presence (in the case of add tag) or absence (in the case of delete tag).
 
-    Use case resumes at step 2. 
+    Use case resumes from step 2.
 
-**Use case: UC05 - Find properties**
+_________________
+
+**Use case: UC05 - Find properties/buyers**
 
 **MSS**
 
-1. `User` requests to find properties by specifying keywords and a set of tags.
-2.  `PropertyWhiz` filters the displayed property list to include only properties that contain the specified keywords in the name and have the specified tags.
+1. `User` requests to find properties/buyers by specifying keywords, tags or price range.
+2. `PropertyWhiz` displays properties/buyers that fulfill the criteria.
 
 Use case ends.
 
-**Use case: UC06 - Import properties**
+_________________
+
+**Use case: UC06 - Import properties/buyers**
 
 **MSS**
 
-1. `User` requests to import properties.
-2. `User` chooses the source csv file.
-3. `PropertyWhiz` imports the rows within the csv file as properties, and prepends the imported properties the displayed property list.
+1. `User` requests to import properties/buyers.
+2. `User` chooses the source CSV file.
+3. `PropertyWhiz` imports the properties/buyers from the source file.
 
 Use case ends.
 
 **Extensions**
 
-* 2a. `User` cancels while choosing a csv file.
-    * 2a1. `PropertyWhiz` shows an error message.
+* 1a. `User` cancels while choosing a CSV file.
+    * 1a1. `PropertyWhiz` shows an error message.
 
     Use case ends.
 
-* 3a. The csv file cannot be opened or read from.
-    * 3a1. `PropertyWhiz` shows an error message.
+* 2a. The CSV file cannot be opened or read from.
+    * 2a1. `PropertyWhiz` shows an error message.
 
     Use case ends.
 
-* 3b. The csv file is incorrectly formatted.
-    * 3b1. `PropertyWhiz` shows an error message.
+* 2b. The CSV file is incorrectly formatted.
+    * 2b1. `PropertyWhiz` shows an error message.
 
     Use case ends.
 
-* 3c. Importing creates duplicate properties within `PropertyWhiz`.
-    * 3c1. `PropertyWhiz` shows an error message.
+* 2c. Importing creates duplicates within `PropertyWhiz`.
```

Comment:
```suggestion
* 2c. Importing causes duplicates within `PropertyWhiz`.
```

Reply:
I think creates sounds better to me, what do the others think? @AY2122S1-CS2103T-W11-4/developers